meson.build

```diff
@@ -0,0 +1,110 @@
+project(
+  'flatpak-builder',
+  'c',
+  license: 'LGPL-2.1-or-later',
+  meson_version: '>=0.49',
+  version: '1.0.11',
+  default_options: '-Dc_std=gnu99',
+)
+
+# TODO: do some cleaning up of useless args that meson already sets by default
+add_project_arguments(
+  '-Werror=empty-body',
+  '-Werror=strict-prototypes',
+  '-Werror=missing-prototypes',
+  '-Werror=implicit-function-declaration',
+  '-Werror=format=2',
+  '-Werror=format-security',
+  '-Werror=format-nonliteral',
+  '-Werror=pointer-arith',
+  '-Werror=init-self',
+  '-Werror=missing-declarations',
+  '-Werror=return-type',
+  '-Werror=overflow',
+  '-Werror=int-conversion',
+  '-Werror=parentheses',
+  '-Werror=incompatible-pointer-types',
+  '-Werror=misleading-indentation',
+  '-Werror=missing-include-dirs',
+  # Necessary for non POSIX stuff in glibc like fallocate
+  '-D_GNU_SOURCE',
+  language: 'c',
+)
+
+inc_dirs = include_directories(
+  '.',
+# FIXME: handle $prefixes/libdwarf/ too or let it to the option? The later would be easier
+  # Dwarf header stuff
+  get_option('dwarf_header_dir'),
+)
+
+cc = meson.get_compiler('c')
+
+# FIXME: this was in the autoconf file but it didn't prevent from configuring. Also, it seems like fdwalk isn't called in the code?!
```

Comment:
This stuff in `configure.ac` might be left over from when flatpak-builder was part of Flatpak.

Reply:
I removed some of it in #387.

Reply:
Yeah, I'll drop the corresponding meson parts that are dropped in your #387 MR then.